api/graphql/spaces/space_types.py

```diff
@@ -9,6 +9,10 @@
 
 
 class DistrictType(PrimaryKeyObjectType):
+    name_fi = graphene.String()
```

Comment:
would it be enough to define these fields in the `Meta` with `get_all_translatable_fields` ?